polkadot/xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -298,54 +360,59 @@ benchmarks! {
 		<T::XcmConfig as xcm_executor::Config>::AssetTrap::drop_assets(
 			&origin,
 			assets.clone().into(),
-			&XcmContext {
-				origin: Some(origin.clone()),
-				message_id: [0; 32],
-				topic: None,
-			},
+			&XcmContext { origin: Some(origin.clone()), message_id: [0; 32], topic: None },
 		);
 
 		// Assets should be in the trap now.
 
 		let mut executor = new_executor::<T>(origin);
 		let instruction = Instruction::ClaimAsset { assets: assets.clone(), ticket };
 		let xcm = Xcm(vec![instruction]);
-	} :{
-		executor.bench_process(xcm)?;
-	} verify {
+		#[block]
+		{
+			executor.bench_process(xcm)?;
+		}
 		assert!(executor.holding().ensure_contains(&assets).is_ok());
+		Ok(())
 	}
 
-	trap {
+	#[benchmark]
+	fn trap() -> Result<(), BenchmarkError> {
 		let mut executor = new_executor::<T>(Default::default());
 		let instruction = Instruction::Trap(10);
 		let xcm = Xcm(vec![instruction]);
 		// In order to access result in the verification below, it needs to be defined here.
 		let mut _result = Ok(());
-	} : {
-		_result = executor.bench_process(xcm);
-	} verify {
-		assert!(matches!(_result, Err(ExecutorError {
-			xcm_error: XcmError::Trap(10),
-			..
-		})));
+		#[block]
+		{
+			_result = executor.bench_process(xcm);
```

Comment:
Here you should avoid the underscore because `result` is used (minor remark)

Reply:
The CI seems to not like it 
https://github.com/paritytech/polkadot-sdk/actions/runs/12048581684/job/33593713973#step:4:51